user/service/client/client.go

```diff
@@ -202,6 +204,23 @@ func (c *Client) Delete(ctx context.Context, id string, deleet *user.Delete, con
 			logger.WithError(err).Error("Unable to destroy profile")
 		}
 	}
+	//Sending delete user event to kafka
+	userName := *result.Username
+	logger.Infof("preparing to send delete user kafka message for %v", userName)
+	if !strings.HasSuffix(userName, "@tidepool.io") && !strings.HasSuffix(userName, "@tidepool.org") {
+		var role []string
+		if result != nil && result.Roles != nil {
+			role = *result.Roles
+		}
+		eventType := "delete-user"
+		userEventMessage := map[string]interface{}{
+			"event": eventType,
```

Comment:
Do you need the event type as part. of the payload? It's already sent as cloud event type.

Reply:
not if I do what I originally had and you suggested here: `This can panic if the payload doesn't have event defined. KafkaMessage(event string, data map[string]interface{}) might a better function signature.`